lib/App.jsx

```diff
@@ -25,6 +25,9 @@ import {
 
 import './global.css'
 
+import 'typeface-nunito'
+import "circular-std";
```

Comment:
Can we change to single quotes and avoid the semi-colon here? I'm surprised the linter didn't catch this